extensions/reactive-pg-client/runtime/src/main/java/io/quarkus/reactive/pg/client/runtime/PgPoolRecorder.java

```diff
@@ -49,6 +51,33 @@ public class PgPoolRecorder {
     private static final TypeLiteral<Instance<PgPoolCreator>> PG_POOL_CREATOR_TYPE_LITERAL = new TypeLiteral<>() {
     };
 
+    private final RuntimeValue<DataSourcesRuntimeConfig> runtimeConfig;
+    private final RuntimeValue<DataSourcesReactiveRuntimeConfig> reactiveRuntimeConfig;
+
+    @Inject
```

Comment:
interesting, does `jakarta.inject.Inject;` has any effect on records at all? I didn't know.

Reply:
It doesn't really. If the annotation wasn't present, it would work exactly the same.

If anything, this is more of a documentation that injection happens. But it's way out of any specification, so perhaps it shouldn't be here after all :shrug:

Reply:
copy that, thanks